study.md

```diff
@@ -34,7 +34,7 @@ including class. Why might Array redefine methods included from Enumerable?
 Please give reasons for the methods you list.
 
 ```md
-<!-- your answer here -->
+`find` and `map`...  I'm not really sure, other than Enumerable seems to just be a more specific version of Array? Enumerable methods are able to access each level in an array (like if it's nested or something), so operations like `find` or `map` are able to look at an array as if it were flattened and therefore can be used with an enumerable. I think.
```

Comment:
`Array` might redefine methods included from `Enumerable` if custom behavior is desired.  For example, `Array` might redefine `each` to only return elements with even indices on each iteration.